studio/components/interfaces/Organization/BillingSettings/BillingAddress/BillingAddress.tsx

```diff
@@ -54,15 +61,15 @@ const BillingAddress = ({ loading, address, onAddressUpdated }: BillingAddressPr
   const onSubmit = async (values: any, { setSubmitting }: any) => {
     try {
       setSubmitting(true)
-      const updatedCustomer = await patch(`${API_URL}/organizations/${orgSlug}/customer`, {
+      const updatedCustomer = await patch(`${API_URL}/organizations/${slug}/customer`, {
```

Comment:
This refactor would've been unnecessary once we have the this PATCH call in a mutation, right? It will invalidate the cache key and the GET query will happen on its own.

Reply:
yup! the PATCH call mutation change is added in https://github.com/supabase/supabase/pull/16347/files